### PR TITLE
add SALESFORCE to gallery + new web trader gif

### PR DESCRIPTION
**Pull request by amadoukane96**

### CODE REVIEW

**Comment by amadoukane96** [comment]
Hey @chriddyp can I merge this?

**Comment by xhlulu** [comment]
Looks good! :dancer:

File: tutorial/gallery.py [hunk 1]
```diff
@@ -139,7 +139,20 @@ def AppSection(app_name,
             PDF reports.
             '''
 
-        )
+        ),
+        
+        AppSection(
+            app_name='Salesforce Dashboard',
+            app_link='https://dash-gallery.plotly.host/dash-salesforce-crm',
```
Comment (xhlulu) [inline tutorial/gallery.py]: Check with Jack if this one should be hosted on Heroku instead of DDS.

Comment (amadoukane96) [reply tutorial/gallery.py]: right! thank you 💯

**Pull request merged**